recipesearch/src/Recipes/SearchBar/SearchBar.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react'
+import './SearchBar.css';
+import  {searchRecipesAction} from './action';
+import { connect } from 'react-redux';
+
+export class SearchBar extends Component {
+    state = {
+        searchValue: '',
+    }
+
+    handleSearchChange = (event) => {
```

Comment:
Don't you mean `handleSearchValueChange`?

Reply:
@edmondatto Noted let me make the relevant changes

Reply:
I have made the relevant changes